static/src/javascripts/projects/common/modules/analytics/google.js

```diff
@@ -34,9 +34,14 @@ define([
         });
     }
 
+    function trackNativeAdLinkClick(slotName, tag) {
+        ga(send, 'event', 'Click', 'Native Ad', tag);
+    }
```

Comment:
Hi,
2 things.

I've created a new dimension for the Ad Slot Name (adSlotName). This can be referenced within here as 'dimension25'.

Also, if could add the non-interaction boolean - "nonInteraction: true"
Although, you could argue this isn't needed in this case (prevent bounce rate being affected by a click) as its an external click, in this case (I believe) the click opens a new tab and so we the user may well continue to browse theguardian.com